tools/ubuntu-setup/docker-xenial.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+#
+#  WARNING:  This is EXPERIMENTAL support for running OpenWhisk on the latest
+#            stable version of Docker CE on Ubuntu Xenial or later.  Proceed
+#            at your own risk.
+#
+#  Currently, ./all.sh does not support running this shell script.ls
+#
+
+set -e
+set -x
+
+sudo apt-get -y install apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+sudo apt-key fingerprint 0EBFCD88
+
+sudo add-apt-repository \
+    "deb [arch=$(uname -m)] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+sudo apt-get -y update -qq
+
+sudo apt-get purge lxc-docker
+sudo apt-cache policy docker-engine
+
+# AUFS
+sudo apt-get -y install "linux-image-extra-$(uname -r)"
+
```

Comment:
I think for AUF do you also need `linux-image-extra-virtual` I think
```
sudo apt-get -y install linux-image-extra-virtual
``

Reply:
Actually, for xenial it looks like you don’t need any form of `linux-image-extra` package; the goodies have been compiled in.